tests/scripts/basic-build-test.sh

```diff
@@ -121,7 +121,7 @@ echo
 echo '################ compat.sh ################'
 {
     echo '#### compat.sh: Default versions'
-    sh compat.sh -m 'tls1 tls1_1 tls1_2 dtls1 dtls1_2'
+    sh compat.sh -m 'tls1 tls1_1 tls12 dtls1 dtls12'
```

Comment:
Why are we passing `-m` at all?

Reply:
We could skip it indeed, though keeping it might make the difference with the next run (`-m ssl3`) slightly more explicit. To be honest, I realised after pushing that I made a different choice in 2.28 and development, but didn't think it was worth pushing again just for that.

Reply:
I'm definitely for not passing `-m` in development. But in the LTS, we aren't going to add a new TLS version, so we only need to get the list right this once. So ok, let's stick with that.